src/main/java/io/appium/java_client/serverevents/ServerEvents.java

```diff
@@ -0,0 +1,19 @@
+package io.appium.java_client.serverevents;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class ServerEvents {
+
+    public final List<CommandEvent> commands;
+    public final List<TimedEvent> events;
+    public final String jsonData;
```

Comment:
should it be string or JSONObject?

Reply:
string, because its only purpose is to write out to disk.